packages/aws-cdk/lib/settings.ts

```diff
@@ -222,6 +212,42 @@ export class Settings {
     return ret;
   }
 
+  private static parseStringContextListToObject(argv: yargs.Arguments) {
```

Comment:
@eladb - what's the position on private static?

Reply:
happy to remove the static if required...

Reply:
Mostly in the code base these types of helper like functions are just defined as `function( ... ) { }` like you see `expandHomeDir()` at the bottom of this class. I honestly don't know if that is a specific decision by the team or just what I've seen.

Reply:
Ok, the reason is because the fromCommandLineArguments is Static, so those functions need to be static. I will specify the return type on those in the next commit.